### PR TITLE
feat(overlays): add overlays system

### LINKED ISSUE #43: overlays
Add overlays into bot

### PR DESCRIPTION
**Pull request by Satont**
Closes #43

### CODE REVIEW

File: public/overlay.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Overlays</title>
+    <script src="/socket.io/socket.io.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.24.0/moment.min.js"></script>
+  </head>
+  <body>
+    <p id="six7urfkCy"></p>
```
Comment (AlexWayfer) [inline public/overlay.html]: What a strange `id`.

File: public/vue/overlays/list.vue [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      else return text
+    }
+  },
+  computed: {
+    location: function () {
+      return window.location.protocol + "//" + window.location.hostname + (window.location.port ? ':' + window.location.port: '');
+    }
+  }
+};
+</script>
```
Comment (AlexWayfer) [inline public/vue/overlays/list.vue]: No newline at the end of file is bad, I think.

I suggest to use editorconfig.org.

File: public/overlay.html (hunk 1 above)
Comment (Satont) [reply public/overlay.html]: Just random salt.

File: public/vue/overlays/list.vue (hunk 2 above)
Comment (Satont) [reply public/vue/overlays/list.vue]: Yep, should be fixed

**Pull request merged**